### ISSUE TITLE
Implement the things in the contributing.md

### ISSUE THREAD

**Issue by krshubham** [krshubham/e2e-socket-chat#1]
Please see the guidelines

Thanks!

**Comment by minibhati93** [comment]
I would like to help.

**Comment by krshubham** [comment]
@minibhati93 Would appreciate your effort! Any ideas about the contributions you can make? I would also like to work on them.

**Comment by minibhati93** [comment]
Thanks @krshubham . I can work on the login screen in the beginning.

**Comment by krshubham** [comment]
That'll be great. Can we have something like simply ask a person a username and then  assign that name to the current socket so that messages given by that person are labelled by his username? Also were you able to setup the whole repo locally?

**Comment by minibhati93** [comment]
Yes, I was able to set up the repo locally though I'm getting the _connection refused_ error. 
It'll take me some time to get familiar with `socket.io` but sure we can assign the username to the current socket. Also, should the username refresh with each session or remain the same for a particular user?
![screen shot 2016-11-01 at 10 36 08 pm 1](https://cloud.githubusercontent.com/assets/5766372/19898992/545c403e-a084-11e6-8991-811e5e307f14.png)

**Comment by krshubham** [comment]
Okay, Just check in the client Js file for the websocket it is getting connected to !
For local setup it should be a local IP address

**Comment by minibhati93** [comment]
Thanks. It is working now. I think we should probably update the readme with the steps as well. Are we supposed to use any `js` libraries like `angular` or `bootstrap` for the `UI`? Also, should the username refresh with each session or remain the same for a particular user?

**Comment by krshubham** [comment]
let the username refresh in each session, and well, we can use bootstrap, where would you use Angular? I mean any great benefits of using angular if you could tell me for a convincing discussion!

**Comment by minibhati93** [comment]
Sure 👍  `bootstrap` would be enough according to me for the `UI`. Angular has its own benefits but I am not sure about the latest update (`Angular 2`).

**Comment by krshubham** [comment]
@minibhati93 any other place to communicate, whatsapp?

**Comment by minibhati93** [comment]
Sure. Write me a mail at bhati.mini93@gmail.com, we can discuss there.

**Comment by codejockie** [comment]
Is this still open?

**Comment by krshubham** [comment]
Hi @codejockie , I appreciate you coming here, but this project is no longer maintained and so if you are interested to work I have one more project named [codeslam](https://github.com/krshubham/codeslam) where you can contribute. I will update the issues in that repo and add hacktoberfest tag!
